packaging/distros/raspbian/Dockerfile

```diff
@@ -6,7 +6,7 @@ ARG BASE_BUILDER
 # Lookup the name to use below but should follow the '<distro>-base' convention with slashes replaced.
 # Use buildkit to skip unused base images: DOCKER_BUILDKIT=1
 
-FROM balenalib/rpi-raspbian:buster AS raspbian-buster-base
+FROM balenalib/rpi-raspbian:bullseye AS raspbian-bullseye-base
```

Comment:
This is actually changing the Raspbian build, I'm not sure we want that as it's not Debian per-se. Not sure why we jumped from buster to bookworm for Raspbian but that's not part of this change.

Please make sure you're not just globally changing any reference to the string.

Reply:
Reverted.